html/feed/feed.js

```diff
@@ -386,7 +386,21 @@ function generateProfileCards(data, container) {
     container.appendChild(card);
   });
 
-  
+
   generatePage();
```

Comment:
the function cant call itself if its never being called in the first place, why is this inside its own function ? 
the page has to generate content so thats why i call it in the global scope : )